torch/distributions/kl.py

```diff
@@ -157,6 +158,15 @@ def _kl_bernoulli_bernoulli(p, q):
     return t1 + t2
 
 
+@register_kl(Binomial, Binomial)
+def _kl_binomial_binomial(p, q):
+    # from https://math.stackexchange.com/questions/2214993/
+    # kullback-leibler-divergence-for-binomial-distributions-p-and-q
+    if p.total_count != q.total_count:
+        raise(NotImplementedError, 'KL between Binomials with different total_count is not implemented')
```

Comment:
That's probably not what you wanted to do. You're raising a tuple as an exception